src/components/SeverityIndicator/SeverityIndicator.tsx

```
@@ -35,22 +35,19 @@ function SeverityIndicatorGenerator({ severity }: { severity: Severity }) {
-      <TooltipContent>{severityToText(severity)}</TooltipContent>
-    </Tooltip>
+    <SimpleTooltip content={severityToText(severity)}>
+      <div className={s.indicator}>
+        {Array.from(new Array(5)).map((_, i) => (
+          <div
+            key={i}
+            className={s.indicatorCell}
+            style={{
+              backgroundColor: i < pivot ? COLORS[i] : 'var(--faint-weak)',
+            }}
+          ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)